### ISSUE TITLE
Any plans on adding motor strength to the component?

### ISSUE THREAD

**Issue by JeroenVanLeusden** [ualex73/slide#16]
Any plans on adding motor strength to the component? On Reddit they posted the endpoint and settings for light/medium with the warning not go go beyond 1450 otherwise you'll brick you device

https://www.reddit.com/r/slidecurtains/comments/1dfv4ix/comment/l94hloi/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

**Comment by ualex73** [comment]
Yes, I am aware of the motor strength, I already put it in the underlying library (goslideapi). I will expose it soon in the HA component.

**Comment by ualex73** [comment]
I submitted the code to the master, I hope it works (I do not get an error on my side)

**Comment by bramwalet** [comment]
Thanks for adding support to set the motor strenght of the slide. I'm the author of the referred reddit comment and wanted to share my approach how I obtained the endpoint and json values. Please see this post (in Dutch) on the GoT forum: https://gathering.tweakers.net/forum/list_message/79294594#79294594

If you want I can also reverse engineer the values for the third motor strength setting. Or, using the method described above, you can do this yourself ;-).

**Comment by ualex73** [comment]
@bramwalet  thanks for sharing the information, I did not do reverse engineer myself (currently my iPhone app does not open :-() and it is bit of work to get a second phone.

**Comment by bramwalet** [comment]
You're welcome!

A second phone won't be necessary. If your computer (Mac?) supports Android Studio, you can run the emulator of Android Studio as a virtual phone running Android. 

You can install the Android Slide app onto that device and pair it with your Slide. 

If you run Mac as your desktop, both Android Studio and Http Toolkit seem supported on that OS.

**Comment by ualex73** [comment]
Light: maxcurrent=900, calib_current=850
Medium: maxcurrent=1250, calib_current=1200
Strong: maxcurrent=1500, calib_current=1450

**Comment by ualex73** [comment]
Strong is added to the integration :-)
